### ISSUE TITLE
Numba deprecation warning

### ISSUE THREAD

**Issue by theodumont** [jni/skan#86]
Thank you for the great software!

When using `skan` 0.8 with python 3.8.3 we get a warning message:
```
NumbaDeprecationWarning: The 'numba.jitclass' decorator has moved to 'numba.experimental.jitclass' to better reflect the
experimental nature of the functionality. Please update your imports to accommodate this change and see
http://numba.pydata.org/numba-doc/latest/reference/deprecation.html#change-of-jitclass-location for the time frame.
@numba.jitclass(csr_spec)
```
caused by this part of the `csr.py` module:
```py
@numba.jitclass(csr_spec)
class NBGraph:
```

Indeed, on [the Numba Website](http://numba.pydata.org/numba-doc/latest/reference/deprecation.html#change-of-jitclass-location) we can read:
> Between versions 0.48 and 0.49 Numba underwent a large amount of refactoring. One of the decisions made by the core developers as part of this refactoring was to move `numba.jitclass` to a new location `numba.experimental.jitclass`. This is to help reinforce expectations over the behaviour and support for certain features by deliberately placing them in an `experimental` submodule.

Replacing `numba.jitclass` by `numba.experimental.jitclass` in `csr.py` would remove this warning. 

```py
@numba.experimental.jitclass(csr_spec)
class NBGraph:
```

Do you want me to address this?

**Comment by jni** [comment]
Thanks for reporting @theodumont ! Indeed it would be best to fix this. If you are comfortable with the GitHub pull request workflow I would definitely appreciate a PR, otherwise I will try to get this done in the next few days.

**Comment by theodumont** [comment]
Great!
I'll make a PR right now, don't hesitate to tell me if something is wrong with it.

**Comment by jni** [comment]
Fixed in #87
